app/models/news/importer.rb

```diff
@@ -0,0 +1,138 @@
+module News
+  class Importer
+    DEFAULT_ITEMS_FILE = Rails.root.join('db/news/govuk_stories.json').freeze
+    NEWS_ITEM_KEYS = %w[
+      headline
+      slug
+      precis
+      story
+      validity_start_date
+      validity_end_date
+      themes
+    ].freeze
+
+    class << self
+      def import!(src_file = DEFAULT_ITEMS_FILE)
+        new(src_file).import!
+      end
+    end
+
+    def initialize(src_file)
+      @src_file = src_file
+    end
+
+    def stories
+      @stories ||= JSON.parse(@src_file.read)['news'] || []
+    end
+
+    def import!
+      raise NotAvailableOnXi if TradeTariffBackend.xi?
+
+      import_time = Time.zone.now
+
+      ::News::Item.db.transaction do
+        create_collections
+
+        stories.each { |story| import_story(story, import_time) }.length
```

Comment:
Do you want to return the number of stories imported?

My understanding is that returns the object it was invoked upon.
therefore `stories.length` is independent of the success/failure of the import stories.

Reply:
any story which fails to import will bomb the transaction - so the number of stories imported _has_ to match the length of `.stories` - it _cannot_ be different?